libraries/RC_Channel/RC_Channel.h

```diff
@@ -184,6 +184,11 @@ class RC_Channel {
         AIRMODE =             84, // enable / disable airmode for copter
         GENERATOR   =         85, // generator control
         TER_DISABLE =         86, // disable terrain following in CRUISE/FBWB modes
+        SCRIPTING_1 =         87, // for use by onboard lua scripting
```

Comment:
I'd really like to see these moved into the 200's, we are running out of room below 100, and I envision these being used not only as positional switches but also as continuous variables in scripts...even thought LUA could read then as is for PWM value

Reply:
instead of this, which I had suggested and @IamPete1 quickly, and generously, coded to get user re-configurations into a script without the user having to actually edit a script (simple, but which may be daunting to some), I think this is a better approach...I am proposing #14787

-    it allows analog variable input, but still can be used to identify an RC channel for switch or real time analog variable input
 -   it does not consume, what is becoming precious RCx_OPTION space, although moving the enum to a 200 class position in this PR would eliminate that as well as providing intuitive continuous RC channel variable input...
-    does not consume code space on 1MB boards like RCx_OPTION would

 flagging both with DEVCALL for discussion

Reply:
moved to 300's